java/run-tests.sh

```diff
@@ -0,0 +1,2 @@
+docker build . -t kensho-auth-java
+docker run -v ${PRIVATE_KEY_FILE}:/home/app/src/private_key.pem -e PRIVATE_KEY_FILE="/home/app/src/private_key.pem" -e SCOPES=${SCOPES} -e CLIENT_ID=${CLIENT_ID} kensho-auth-java
```

Comment:
nit: just using -e NAME should get the value from environment without explicitly using =$NAME

(more important when passing secrets to avoid logging into console, but these are not secrets)

Reply:
I'll change that up in the pr i make for adding c# tests.